articles/migration/updating_versions.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Updating Versions
+description: A guide on updating MonoGame when a new release is pushed.
+---
+
+When a new release of MonoGame is pushed, developers will need to manually update their development environment and/or existing project to make use of the new version.  This guide is intended to walk you through the steps of performing these updates.
+
+# Update dotnet Templates
+Developers will need to update the MonoGame C# Templates used to create new projects.  Doing this depends on if you are using Visual Studio 2022 or through the dotnet cli.
```

Comment:
Requires a note to point out that existing projects are unaffected, but they can still update the nugets to new versions if they wish to.

A new project isn' the only way to get the latest :D

Reply:
Added note